### ISSUE TITLE
Implementation of counting sort is missing

### ISSUE THREAD

**Issue by varunkmr19** [luiseduardo1/Cormen-algorithms-implementation#3]
I would love to add some of the missing algorithms like counting sort, shell sort, etc.

**Comment by ArnabBanik-repo** [comment]
Can I take up this issue and contribute?

**Comment by luiseduardo1** [comment]
> Can I take up this issue and contribute?

Yes, sure! The project is no longer really up to date, but feel free to contribute to it!
